module/scripts/index/files-importing-controller.js

```diff
@@ -355,3 +363,27 @@ Refine.FilesImportingController.prototype._createProject = function() {
     );
   });
 };
+
+Refine.TagsManager = {};
+Refine.TagsManager.allProjectTags = [];
```

Comment:
Ouch, that's also content copied from OpenRefine, right? Ideally, this should be exposed by OpenRefine in a way that lets it be reused by extensions, so they don't have to vendor this code…

Reply:
What do i need to do here?

Reply:
Nothing in this PR, it's just a comment. I assume you're doing this because the tag manager isn't defined by OpenRefine core in this context.

Reply:
That is correct